react/components/IconPack.js

```diff
@@ -1103,6 +1103,43 @@ const IconPack = () => {
             fill="currentColor"
           />
         </g>
+
+        {/** ====== Social Networks ====== */}
```

Comment:
I think this is a brand or inf icon, We really need another category?

Reply:
Brand would be good. Nice idea.